packages/create-package/README.md

```diff
@@ -6,17 +6,37 @@
 
 Provides an `npm init` script for creating new "UI5 Web Components" packages.
 
-## Usage
+## Usage with npm
 
-with `npm`
+```
+Usage:
 
-- `npm init @ui5/webcomponents-package` (the user will be asked for **name**, **port**, **tag**);
-- `npm init @ui5/webcomponents-package <NEW-PACKAGE-NAME>` (the user will be asked for **port** and **tag** only).
+# npm 6.x
+    npm init @ui5/webcomponents-package [OPTIONS]
+# npm 7+, extra double-dash is needed:
+    npm init @ui5/webcomponents-package -- [OPTIONS]
 
-And with `yarn`
+Options:
+    --name <string>     - defines package name
+    --tag <string>      - defines predefined custom element tag name

```

Comment:
defines the tag name of the sample web component that will be created in your new package